howto/cluster/appliance.md

```diff
@@ -0,0 +1,47 @@
+If you're running the Anbox Cloud Appliance and you want to extend the available container capacity, you can join additional machines to the appliance.
+
+The following instructions describe how to add a machine. Also see [Multi-node support for the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/capacity-planning/17765#clustering-anbox-cloud).
+
+## Allow internal network communication
+
+To allow communication between the different machines, you must configure the internal network that the machines use to allow incoming traffic.
+
+How to do this depends on your network configuration. On AWS, for example, you can modify the security group attached to the instance to allow all incoming traffic on the internal subnet that it sits on:
+
+![AWS security group](images/appliance_multi_node_aws_ports.png)
+
+In this example, incoming traffic is allowed when it originates in the internal subnet `172.31.16.0/20`.
```

Comment:
I actually made a mistake here. The rule has to allow traffic from 172.31.0.0/16 

Updated screenshot is

![image](https://user-images.githubusercontent.com/433752/174242506-b589d697-5493-42cd-9ec1-4319bee38ae1.png)